examples/nlp/dialogue_state_tracking/dialogue_state_tracking_sgd.py

```diff
@@ -153,7 +150,19 @@
 )
 
 parser.add_argument(
-    "--enable_pin_memory", action="store_true", help="Enabled the pin_memory feature of Pytroch's DataLoader",
+    "--enable_pin_memory", action="store_true", help="Enables the pin_memory feature of Pytroch's DataLoader",
+)
+
+parser.add_argument(
+    "--state_tracker", type=str, default='baseline', help="Specifies the state tracker mode",
+)
+
+parser.add_argument(
+    "--log_data_warnings", action="store_true", help="Enables logging warnings for data",
+)
+
+parser.add_argument(
+    "--eval_debug", action="store_true", help="Enables logging evaluation info for debugging",
```

Comment:
can we just use logging.debug for both --log_data__warning and --eval_debug?

Reply:
Done